### DISCUSSION TITLE
Weekly community call

### DISCUSSION THREAD

**Discussion by luke-feeney** [terminusdb/terminusdb#1064] [General]
What about a community call once a week? Could be formatted like the office hour of before - could do it on the stage in the Discord? 

Format would be just 30 mins and attendance who is around/available?

Could include a standard 5/10 minute intro to the project and repos each week for any newbies.

**Comment by AbhijithGanesh** [comment]
I would like this to be a 30 minute meeting where members and contributors can show and tell, discuss their progress, debate decisions, discuss issues and standard and so on.

**Comment by luke-feeney** [comment]
this is a great suggestion - lets have a first one to discuss putting this stuff in the following meetings.

**Comment by AbhijithGanesh** [comment]
Very good call out, I completely agree

**Comment by luke-feeney** [comment]
I think we will start this next week. Time that we are thinking is 3pm CET on Wednesdays. 

Plan is show and tell, feedback, and discussion of priorities/issues.

**Comment by AbhijithGanesh** [comment]
Great, now can this be moved to discussions?

**Reply by luke-feeney** [reply]
yes!
